build.gradle

```diff
@@ -25,6 +33,12 @@ buildscript {
     }
 }
 
+
+plugins {
```

Comment:
Not 100% sure, but this could be removed since it is an alternative syntax of `apply plugin` that is done in the if jdk >= 11